File: OpenSilver.Samples.TelerikUI/OpenSilver.Samples.TelerikUI/Other/Internal/ILazyAssemblyLoader.cs
using System.Collections.Generic;
using System.Reflection;
using System.Threading.Tasks;

namespace OpenSilver.Samples.TelerikUI
{
    public interface ILazyAssemblyLoader
    {
        Task<IEnumerable<Assembly>> LoadAssembliesAsync(IEnumerable<string> assembliesToLoad);
    }
}

File: OpenSilver.Samples.TelerikUI/OpenSilver.Samples.TelerikUI/Other/Internal/ILazyAssemblyLoader.vb
Imports System.Collections.Generic
Imports System.Reflection
Imports System.Threading.Tasks

Namespace OpenSilver.Samples.TelerikUI
    Public Interface ILazyAssemblyLoader
        Function LoadAssembliesAsync(assembliesToLoad As IEnumerable(Of String)) As Task(Of IEnumerable(Of Assembly))
    End Interface
End Namespace
